src/components/EventEdit.tsx

```diff
@@ -154,7 +158,16 @@ class EventEdit extends React.PureComponent<PropsType> {
   public toggleAllDay() {
     this.setState({ allDay: !this.state.allDay });
   }
+  public toggleRecurring() {
+    const value = this.state.rruleOptions ? undefined : { freq: 'WEEKLY', interval: 1 };
+    this.setState({ rruleOptions: value });
+  }
+
 
+  public handleRRuleChange(rrule: RRuleOptions): void {
+    this.setState({ rruleOptions: rrule });
+    console.log(rrule);
```

Comment:
`console.log`... Please remove. :)
I'd make it a habit not to commit those, not even in a WIP branch.

Reply:
yeah it definitely didn't get committed on purpose